pages/mydoc/faq_keycult_faq.md

```diff
@@ -99,6 +103,24 @@ This is meant to be a collection of commonly asked questions and their answers.
                         </div>
                     </div>
                     <!-- /.panel -->
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <h4 class="panel-title">
+                                <a class="noCrossRef accordion-toggle" data-toggle="collapse" href="#collapseEightteen">Will X PCB fit a Keycult keyboard?</a>
+                            </h4>
+                        </div>
+                        <div id="collapseEightteen" class="panel-collapse collapse noCrossRef">
+                            <div class="panel-body">
+                                The recommended supported PCBs are the ones sold on the Keycult store at <a href="https://keycult.com/collections/accessories" target="_blank">here</a>. The following PCBs will also fit as well.
+                                <ul>
+                                    <li>No. 1/60: WT60-G</li>
+                                    <li>No. 1/65 or No. 2/65: WT65-G</li>
+                                    <li>TKL: WT80-G (Note: This PCB will NOT work on the original No. 1)</li>
```

Comment:
we can say for OG No. 1 they can refer to this [list](https://docs.google.com/spreadsheets/d/1GpM--fKeDw_tIDZj4m2jmbMavSyXeOu3QHzfTJwz-IM/edit#gid=0)